features/step_definitions/lolcommits_steps.rb

```diff
@@ -73,7 +73,7 @@
 end
 
 Then /^I should be (prompted for|presented) "(.*?)"$/ do |_, prompt|
-  assert @stdout.read.to_s.include?(prompt)
+  expect(@stdout.read.to_s).to include(prompt)
```

Comment:
Yeah, exactly.  In theory we should be able to mixin MiniTest assertions the same way (although it requires a bit of boilerplate: https://github.com/cucumber/cucumber/wiki/Using-MiniTest) but that doesn't actually function if rspec functions are also loaded due to E_TOOMUCHMAGIC of different frameworks trying to do the right thing automatically.

That said even if it did work it introduces a larger amount of boilerplate than the 3-4 one line assertions.

I think its unlikely to be able to strip rspec-expectations out of Aruba, I think it uses a lot of its advanced functionality to write its ruleset.  It may be possible to rewrite them all with minitest 5, which would be a pretty epic pull request for Aruba, but I suspect it would be quite an endeavor.